client/src/dataset/addtoproject/DatasetAdd.present.js

```diff
@@ -118,26 +153,30 @@ function DatasetAdd(props) {
     suggestionInput = <div><Loader size="14" inline="true" />{" "}Loading projects...</div>;
   }
 
+  if (!props.dataset) return null;
+
+  const formToDisplay = !isNewProject ?
+    (
+      <div className="mt-4">
+        <form onSubmit={onSubmit} className={"mt-2"}>
+          {suggestionInput}
+          {statusImportDataset}
+          {addDatasetButton}
+        </form>
+      </div>
+    ) : null;
+
+  const header = HeaderAddDataset(props.dataset);
   return (
-    <Modal isOpen={props.modalOpen} toggle={props.closeModal}>
-      <ModalHeader toggle={props.closeModal}>
-        Add dataset to existing project
-      </ModalHeader>
-      <ModalBody className={"text-break"}>
-        <Row className="mb-3">
-          <Col>
-            <form onSubmit={onSubmit}>
-              {suggestionInput}
-              {statusImportDataset}
-            </form>
-          </Col>
-        </Row>
-      </ModalBody>
-      <ModalFooter>
-        {closeButton}
-        {addDatasetButton}
-      </ModalFooter>
-    </Modal>
+    <>
+      <Row className="mb-3">
+        <Col sm={10} md={9} lg={8} xl={7}>
+          { header }
```

Comment:
If HeaderAddDataset is a component, as suggested above, you can do:
```suggestion
          <HeaderAddDataset dataset={props.dataset} />
```

Reply:
sure. updated to be component :)